### ISSUE TITLE
 oraclejdk8 build failing - No compiler is provided in this environment

### ISSUE THREAD

**Issue by dturczanski** [Shippable/support#4603]
Build passing locally, no changes in shippable config. Fails since Friday.

https://app.shippable.com/bitbucket/digitaltoucan/jql-search-extensions/runs/103/1/console
https://app.shippable.com/bitbucket/digitaltoucan/structure-cloud/runs/70/1/console

**Comment by dturczanski** [comment]
It started working now again... What was the problem?

**Comment by ric03uec** [comment]
@dturczanski we'd made some changes to the way the builds switch Java version in the latest release. We noticed some customers facing issues with their builds especially the ones using maven. We've reverted the Java switch logic now because of which your jobs are running correctly. We're closely monitoring this and working on a correct  fix for the initial issue we were trying to fix. We apologise for the inconvenience.
